paddle/scripts/docker/build.sh

```diff
@@ -49,29 +49,27 @@ cmake .. \
       -DCUDNN_ROOT=/usr/ \
       -DWITH_STYLE_CHECK=${WITH_STYLE_CHECK:-OFF} \
       -DWITH_TESTING=${WITH_TESTING:-OFF} \
-      -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+      -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+      -DUNITTEST_USE_VIRTUALENV=OFF
 
 cat <<EOF
-========================================
-Building in /paddle/build ...
+============================================
+Building and installing in /paddle/build ...
```

Comment:
If we do not use `virtualenv` for Python unit-test in Docker. Then we must install Paddle into that container. So we cannot run Python tests without `make install`.

Also, the script before split `make` and `make install` in two steps, but always `make install` later. So just call `make install` once is simpler.